BuildFFMpeg.ps1

```diff
@@ -0,0 +1,37 @@
+param (
+    [ValidateSet('x64', 'x86', 'ARM')]
+    [string] $Platform = 'x64',
+
+    [string] $MSys2Dir = 'C:\msys64',
+
+    [string] $VSInstallDir = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2017\Enterprise",
+    [string] $VCVersion = '14.16.27023',
+    [string] $VCInstallDir = "$VSInstallDir\VC\Tools\MSVC\$VCVersion",
+
+    [string] $WindowsKitsDir = "${env:ProgramFiles(x86)}\Windows Kits",
+    [string] $UniversalCRTSdkDir = "$WindowsKitsDir\10",
+    [string] $UCRTVersion = '10.0.17763.0',
+    [string] $NetFxVersion = '4.7.2'
```

Comment:
This should be generic and either be passed into the script or lookup the latest installed SDK.